apps/nextjs/src/components/icons/picker/icon-picker.tsx

```diff
@@ -35,33 +35,48 @@ export const IconPicker = ({ initialValue, onChange, error, onFocus, onBlur }: I
 
   const groups = notNullableData.map((group) => {
     const options = group.icons.map((item) => (
-      <Combobox.Option value={item.url} key={item.id}>
-        <Group>
-          <Image src={item.url} w={20} h={20} />
-          <Text>{item.name}</Text>
-        </Group>
-      </Combobox.Option>
+      <Card
+        onClick={() => {
+          const value = item.url;
+          setValue(value);
+          setPreviewUrl(value);
+          setSearch(value);
+          onChange(value);
+          combobox.closeDropdown();
+        }}
+        key={item.id}
+        p="sm"
+        pos="relative"
+        style={{
+          overflow: "visible",
+          cursor: "pointer",
+        }}
+      >
+        <Box w={25} h={25}>
+          <Image src={item.url} w={25} h={25} radius="md" />
+        </Box>
+        {item.url.endsWith(".svg") && (
+          <Badge pos="absolute" top={0} right={0} style={{ transform: "translate3d(5px, -5px, 0)" }} size="sm">
+            SVG
+          </Badge>
+        )}
```

Comment:
I still prefer the usage of a indicator that already has it's placing implemented and can be disabled with the disabled prop